core/primitives/src/time.rs

```diff
@@ -1,35 +1,51 @@
-use std::default::Default;
-
+/// Provides structs used for getting time.
+///
```

Comment:
I'd also mention that people should be using Clock whenever possible - and write some guideline on when they should use now() vs instant()

Reply:
@mm-near My personal opinion on this is that we should add a large disclaimer, that the usage of `Clock` is incomplete.
And, it should be discouraged. 

```
   let i = Clock::instant();
   /// This get's current time, not mocked time
   i.elapsed().
```

Reply:
```
/// WARNING WARNING WARNING
/// WARNING WARNING WARNING
/// Use at your own risk. The implementation is not complete, we have a places in code not mocked properly.
/// For example, it's possible to call `::elapsed()` on `Instant` returned by `Instant::now()`.
/// We use that that function, throughout the code, and the current mocking of `Instant` is not done properly.
```